src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
+
+  alive() {
+    Animal.alive.filter((creature) => creature.health > 0);
```

Comment:
The `alive` method is intended to update the `Animal.alive` array by removing animals with health less than or equal to zero. However, the current implementation only filters the array without updating it. Consider reassigning the filtered array back to `Animal.alive`.